app/CocoaPods/CPSidebarDocumentsController.swift

```diff
@@ -89,8 +88,8 @@ class CPSidebarDocumentsController: NSObject {
     openPodfileView.frame = documentScrollView.frame
     documentScrollView.superview?.addSubview(openPodfileView)
 
-    // Make sure that you can't tap change the doc types that will do nothing
-    buttons.forEach { self.enableButton($0, select: false) }
+    // Make sure that you can't change the doc types (it will do nothing)
+    buttons.forEach { self.enableButton($0, select: true) }
```

Comment:
won't this enable them to be clicked even though the have not content? ( if not, I might have that function  backwards )